### ISSUE TITLE
META: What is the proper name of stuff?

### ISSUE THREAD

**Issue by ramirezfranciscof** [aiidateam/aiida-tutorials#372]
So my question is: what is the proper nomenclature for the hierarchies of the tutorial? For example:

- [Workflows](https://aiida-tutorials.readthedocs.io/en/tutorial-2021-intro-week_a/sections/workflows/index.html)
  - [Work chains](https://aiida-tutorials.readthedocs.io/en/tutorial-2021-intro-week_a/sections/workflows/workchain.html)
    - [Constructing our first work chain](https://aiida-tutorials.readthedocs.io/en/tutorial-2021-intro-week_a/sections/workflows/workchain.html#constructing-our-first-work-chain)
      - [Define method](https://aiida-tutorials.readthedocs.io/en/tutorial-2021-intro-week_a/sections/workflows/workchain.html#define-method)

One option would be:

- (Tutorial) Section
  - Module
    - (Module) Section
      - (Module) Subsection

Subsections are fine because only (Module) sections have subsections, but it is a bit confusing to have (tutorial) sections and (module) sections. BTW this nomenclature is my best guess, but perhaps you are already using a better way of referencing parts that I was not aware of, that is why I tagged this as a question (at least initially).

This is relevant for when we are referencing other parts of the tutorial in the instructions (what does "as you saw in the previous *section*" refer to?)

**Comment by mbercx** [comment]
> Subsections are fine because only (Module) sections have subsections, but it is a bit confusing to have (tutorial) sections and (module) sections. 

I agree, I've already struggled with this myself a bit. I think we should rename the top level "Section". I was thinking "Topic", but this is dangerously close to the "Topics" _section_ in the tutorial. Maybe "Chapter", but that sound a little too formal. Other suggestions: Unit, Subject, Theme, ...

> This is relevant for when we are referencing other parts of the tutorial in the instructions (what does "as you saw in the previous section" refer to?)

Indeed. One way to refer to a top-level "Section" would be to say e.g.:

_as you saw in the modules on "Running processes"_

**Comment by mbercx** [comment]
I'm starting to get used to calling them "Units" right now:

- Unit
  - Module
    - Section
      - Subsection

This is also the convention I've used in the changes for https://github.com/aiidateam/aiida-tutorials/pull/381.
